Patch created by brosx "diffPackage" tool

usage:
    diffPackage <tarName> <pkgName> <outName> <patchName> <files>

actual:
    diffPackage "busybox-1.27.2.tar.bz2" "busybox" "busybox" "hardcode-libcurses-ld-flags" "\
        scripts/kconfig/lxdialog/Makefile \
    ";


===============================================================================
--- a/scripts/kconfig/lxdialog/Makefile	2017-07-06 17:14:57.000000000 +0200
+++ b/scripts/kconfig/lxdialog/Makefile	2018-03-16 23:26:21.000000000 +0100
@@ -6,13 +6,15 @@
 # Use reursively expanded variables so we do not call gcc unless
 # we really need to do so. (Do not call gcc as part of make mrproper)
 HOST_EXTRACFLAGS = $(shell $(CONFIG_SHELL) $(check-lxdialog) -ccflags)
-HOST_LOADLIBES   = $(shell $(CONFIG_SHELL) $(check-lxdialog) -ldflags $(HOSTCC))
+#HOST_LOADLIBES   = $(shell $(CONFIG_SHELL) $(check-lxdialog) -ldflags $(HOSTCC))
+HOST_LOADLIBES   = -lcurses
 
 HOST_EXTRACFLAGS += -DLOCALE
 
 PHONY += dochecklxdialog
 $(obj)/dochecklxdialog:
-	$(Q)$(CONFIG_SHELL) $(check-lxdialog) -check $(HOSTCC) $(HOST_EXTRACFLAGS) $(HOST_LOADLIBES)
+	#$(Q)$(CONFIG_SHELL) $(check-lxdialog) -check $(HOSTCC) $(HOST_EXTRACFLAGS) $(HOST_LOADLIBES)
+	$(CONFIG_SHELL) $(check-lxdialog) -check $(HOSTCC) $(HOST_EXTRACFLAGS) $(HOST_LOADLIBES)
 
 hostprogs-y	:= lxdialog
 always		:= $(hostprogs-y) dochecklxdialog
